src/app/App.js

```diff
@@ -45,4 +48,24 @@ export default class App extends Component {
     }
   }
 
-}
\ No newline at end of file
+  filterMovies = (query) => {
+    //Want to search by title, overview, and genre
+    const filteredList = this.state.movies.filter(movie => movie.title.includes(query))
+    // console.log(filteredList)
+    // if(filteredList.length === 0) {
+    //   this.setState({ error: 'No results' })
+    // } else {
+    //   this.setState({ searchResults: filteredList })
+    // }
+    this.setState({ searchResults: filteredList })
+  }
```

Comment:
I think we can make this conditional work for us with a few changes. The way I see it, there are three conditions here:
1.) If filteredList.length === 0 and the search bar is empty, then show all movies
2.) If filteredList.length === 0 and the search bar is NOT empty, then set App state error to show "No results"
3.) If filteredList.length > 0, set App state searchResults to filteredList...